log4j-api/src/main/java/org/apache/logging/log4j/Logger.java

```diff
@@ -80,59 +80,61 @@
      * Logs an exception or error that has been caught to a specific logging level.
      *
      * @param level The logging Level.
-     * @param t The Throwable.
+     * @param throwable the Throwable.
      */
-    void catching(Level level, Throwable t);
+    void catching(Level level, Throwable throwable);
 
     /**
-     * Logs an exception or error that has been caught. Normally, one may wish to provide additional information with an
-     * exception while logging it; in these cases, one would not use this method. In other cases where simply logging
-     * the fact that an exception was swallowed somewhere (e.g., at the top of the stack trace in a {@code main()}
-     * method), this method is ideal for it.
+     * Logs an exception or error that has been caught at the {@link Level#ERROR ERROR} level.
```

Comment:
Simpler to say "Logs a Throwable..." here IMO since the argument IS a Throwable and I can define a subclass that is neither an Exception or Error.

Reply:
Corrected.